### ISSUE TITLE
Mod Leggings with Speed decrease FOV

### ISSUE THREAD

**Issue by N1ch0l2s** [SleepyTrousers/EnderIO#5195]
#### Issue Description:
The Dark (and possibly End Steel) Leggings with Speed applied to them seem to decrease FOV upon being worn. Originally I had thought this was happening because my FOV was being increased from my Loot Slash Conquer Agility attribute, which increases movement speed. However I went into a test world and saw that was not the case. If anything, I think LSC's Agility attribute doesn't even increase FOV. I do not know if the FOV decrease is intentional.

#### What happens:
Dark/End Steel Leggings with Speed decrease FOV when equipped. Even with mods like NoFov this is noticeable. Effect is more drastic with higher Speed levels. 

#### What you expected to happen:
Dark/End Steel Leggings with Speed should increase FOV.

#### Steps to reproduce:

1. Get an empowered set of EIO Leggings and put the Speed upgrade on them.
2. Wear them.
3.  Notice your FOV change. 
...

____
#### Affected Versions (Do *not* use "latest"): 1.12.2

- EnderIO: 5.0.50
- EnderCore: 0.5.65
- Minecraft: 1.12.2
- Forge: 14.23.5.2838
- Single Player 

I've attached a gif demonstrating the issue.
![dark steel legig](https://user-images.githubusercontent.com/32824550/62617243-4c9e7800-b8c6-11e9-84a7-330c7da62519.gif)
 demonstrating the effect.

**Comment by N1ch0l2s** [comment]
Before linking it, I have checked Issue #4216. Looking through the configs with various keywords brought me to nothing related to disabling fov correction, unless I am passing by the config option that does that.

**Comment by HenryLoenwind** [comment]
The "zoom in" effect is applied by vanilla, not us. Our code tries to compensate for it, but it seems you have another mod installed that doesn't properly handle the FOV event.

**Comment by LumberWizard** [comment]
I've had that issue before, with Numina. Disable its FOV compensation in its config.

**Comment by HenryLoenwind** [comment]
There's one tiny thing we can do, see commit. This would stop most mods to change the value we set (which makes sense because that value already includes other mod's adjustments).

**Comment by N1ch0l2s** [comment]
> 
> 
> I've had that issue before, with Numina. Disable its FOV compensation in its config.

 I do not have Numina installed.

**Comment by mindforger** [comment]
but there are probably other mods messing with the FoV, check if there are similar configs in their options or test without the suspected mod

you only noted the EIO Versions but not the whole modpack/modlist, so you would be the only one who can nail it down

by the way, the messing with the FoV is not specifically bound to happen, just because the suspected mod applys an effect, it could be present permanently, so just make new worlds and remove a bunch of suspects until the effect goes away and report back with your findings ^^

**Comment by HenryLoenwind** [comment]
I'd also recommend to test with the latest dev build of Ender IO. That way your findings will keep their worth for much longer...

**Comment by N1ch0l2s** [comment]
> 
> 
> but there are probably other mods messing with the FoV, check if there are similar configs in their options or test without the suspected mod
> 
> you only noted the EIO Versions but not the whole modpack/modlist, so you would be the only one who can nail it down
> 
> by the way, the messing with the FoV is not specifically bound to happen, just because the suspected mod applys an effect, it could be present permanently, so just make new worlds and remove a bunch of suspects until the effect goes away and report back with your findings ^^

Good point. Gonna go ahead and test this with only EIO.

**Comment by N1ch0l2s** [comment]
> 
> 
> I'd also recommend to test with the latest dev build of Ender IO. That way your findings will keep their worth for much longer...

Where do I get the latest dev build?

**Comment by HenryLoenwind** [comment]
on our Discord or at https://ci.tterrag.com/job/EnderIO-Modules/job/EnderIO-Hourly/1033/

**Comment by N1ch0l2s** [comment]
Mind I ask how to install it?

**Comment by N1ch0l2s** [comment]
> 
> 
> but there are probably other mods messing with the FoV, check if there are similar configs in their options or test without the suspected mod
> 
> you only noted the EIO Versions but not the whole modpack/modlist, so you would be the only one who can nail it down
> 
> by the way, the messing with the FoV is not specifically bound to happen, just because the suspected mod applys an effect, it could be present permanently, so just make new worlds and remove a bunch of suspects until the effect goes away and report back with your findings ^^

Well it seems that other mods are actually at fault here. I made a Minecraft instance with only the latest EnderIO and EnderCore on Curseforge and this was the result. 
![FOV bug not happening with only EnderIO and EnderCore](https://user-images.githubusercontent.com/32824550/62766684-e93c5380-ba47-11e9-8e83-6ad200be6ede.gif)

So because of that, here is the mod list of my custom pack in which I have FOV problems. 
![Mod List of custom pack](https://user-images.githubusercontent.com/32824550/62766842-391b1a80-ba48-11e9-82f6-3d8c8fa00983.PNG)
This WAS happening without NoFov, too.

**Comment by N1ch0l2s** [comment]
Upon testing it seems having TF2 Stuff Mod and Techguns at the same time is what is causing this. I had installed AOA, TF2 Stuff Mod, and Squake beforehand and nothing was happening until I had Techguns. However upon removing everything but EIO and Techguns, no FOV bug was occuring once again. I decided to take a shot in the dark and added back TF2 stuff mod. Upon testing, the FOV bug was happening once more. 
![The Culprit](https://user-images.githubusercontent.com/32824550/62770687-5e138b80-ba50-11e9-8437-14a213f03f27.gif)

> 
 just make new worlds and remove a bunch of suspects until the effect goes away and report back with your findings ^^
Thank you mindforger for coming up with this. Glad to know it is not actually Ender IO's fault. I am not exactly sure whether I should take this to Rafradek or pwn3d_1337, or both.
